docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc

```diff
@@ -74,10 +78,14 @@ spec:
       vlan: 22
 EOF
 ----
++
+* When `neutron-sriov-nic-agent` is running on the existing Compute nodes, the physical device mappings match the values that are defined in the `OpenStackDataPlaneNodeSet` custom resource (CR). For more information, see xref:pulling-configuration-from-tripleo-deployment_adopt-control-plane[Pulling the configuration from a {OpenStackPreviousInstaller} deployment].
 
-* When `neutron-sriov-nic-agent` is running on the existing Compute nodes, check the physical device mappings and ensure that they match the values that are defined in the `OpenStackDataPlaneNodeSet` custom resource (CR). For more information, see xref:pulling-configuration-from-tripleo-deployment_reviewing-configuration[Pulling the configuration from a {OpenStackPreviousInstaller} deployment].
-
-* Define the shell variables necessary to run the script that runs the fast-forward upgrade. Omit setting `CEPH_FSID`, if the local storage backend is going to be configured by Nova for Libvirt. The storage backend cannot be changed during adoption, and must match the one used on the source cloud:
+* The shell variables that are necessary to run the script that runs the fast-forward upgrade are defined:
```

Comment:
Maybe: "The shell variables for the script that runs the fast-forward upgrade are defined:"
or 
"The shell variables in the script that runs the fast-forward upgrade are defined:"

Why does this line end in a colon? It leads into that note by the looks of it.